clients/js/README.md

```diff
@@ -25,3 +25,18 @@ The following command prints the help menu:
 ```bash
 > node producer.js -h
 ```
+
+## producer_avro.js
+
+This script performs the following tasks:
+1. Registers an Avro schema in the schema registry
+2. Retrieves the same Avro schema from the schema registry
+3. Uses the admin client to create a topic
+4. Reads a csv file line-by-line and uses the Avro schema to serialise the
+   messages before sending them to the topic
+5. Consumes the topic and uses the Avro schema to deserialise the messages
+   before printing them to the console
+
+```bash
+> node producer_avro.js -b localhost:19092 -r http://localhost:18081
```

Comment:
when i run this command, i get error:
```console
bash$ node producer_avro.js -b localhost:19092 -r http://localhost:18081
file:///home/a/src/redpanda-examples/clients/js/producer_avro.js:64
var response = await fetch(publishUrl, {
               ^^^^^

SyntaxError: Unexpected reserved word
    at Loader.moduleStrategy (internal/modules/esm/translators.js:133:18)
    at async link (internal/modules/esm/module_job.js:42:21)
```

Reply:
What version of nodejs are you using? Top-level await has been available since `v14.8` (earlier version supported but you had to use a flag), and the following line ensures the project allows async/await in any `*.js` file:
https://github.com/redpanda-data/redpanda-examples/blob/ea6067aa03e19de001fe198018cc166c8a2721c2/clients/js/package.json#L2

But maybe these details could be included in the README. Let me know if this helps with your situation or not.